resources/default-config.kdl

```diff
@@ -3,6 +3,14 @@
 
 input {
     keyboard {
+        // Control the repeat rate of the keyboard.
+
+        // Repeat delay in milliseconds.
+        repeat-delay 200
```

Comment:
Are you sure about this 200? [Sway](https://github.com/swaywm/sway/blob/b3519c2d2f269f34ff3712ebf1903905a56541a6/sway/input/keyboard.c#L992) and [wlroots](https://github.com/swaywm/sway/blob/b3519c2d2f269f34ff3712ebf1903905a56541a6/sway/input/keyboard.c#L992) default to 600. My GNOME install has 500 (I haven't changed it).

Reply:
Hm, yeah, it could be actually 600. I just remember that we picked the 200 for sctk/anvil at some point, but I don't remember why 200 exactly, maybe I've pushed all of that in the end.

It's 660 in the Xorg actually, but the 25 value is accurate.